doc/external-access.md

```diff
@@ -238,3 +238,31 @@ IP:                       10.100.200.183
 LoadBalancer Ingress:     10.247.108.104
 . . .
 ```
+# Exposing Segmentstore Service on single IP address and Different ports
+
+For Exposing SegmentStoreservices on the same I/P address we will use MetalLB,
```

Comment:
Same comments on `IP address` vs `I/P address`

Reply:
`For Exposing SegmentStoreservices on the same I/P address we will use MetalLB.` The line currently ends in a `,`

Reply:
Done.